components/usage/config.json

```diff
@@ -1,5 +1,9 @@
 {
   "controllerSchedule": "1h",
+  "creditsPerMinuteByWorkspaceClass": {
+    "default": 0.1666666667,
+    "gitpodio-internal-xl": 0.3333333333
```

Comment:
Is this the final name for the workspace class? I think it was agreed that the workspace classes will be called `standard` and `large`.

Reply:
Those are only human names, but the identifier is what's listed here